resources/definitions/os_detection/monitorapp-aiwaf.yaml

```diff
@@ -0,0 +1,20 @@
+os: monitorapp-aiwaf
+text: 'MONITORAPP AIWAF'
+type: appliance
+icon: monitorapp-aiwaf
+group: monitorapp
+
+over:
+    - { graph: device_bits, text: 'Device Traffic' }
+    - { graph: device_processor, text: 'CPU Usage' }
+    - { graph: device_mempool, text: 'Memory Usage' }
+
+discovery:
+    -
+        sysDescr_regex:
+            - '/^Linux /'            # old version
+            - '/^Linux .*#*-AIOS/'
+        snmpwalk:
+            oid: NET-SNMP-EXTEND-MIB::nsExtendCommand
```

Comment:
This doesn't exist (1.3.6.1.4.1.8072.1.3.2.2.1.2) in your test data, even if it did, I'm not sure we'd be wanting to walk an entire table for OS discovery. I'd say drop support for the old version and remove the `- '/^Linux /'            # old version` line and just leave the other one.